packages/web3-modal/src/providers/types.ts

```diff
@@ -133,6 +133,11 @@ export type PstlWeb3ModalOptions<chains extends ReadonlyChains = ReadonlyChains>
      * @description appType is detected and set automtically elsewhere. Escape hatch
      */
     appType?: AppType
+    /**
+     * @name customSafeUrl
+     * @description Custom Safe URL. Useful for testing.
+     */
+    customSafeUrl?: string
```

Comment:
Maybe this would be better one level outside of escapeHatches (inside options)? Doesn’t seem like an escapeHatch and more a configuration